pkg/lib/authn/authenticator/password/deps.go

```diff
@@ -26,6 +26,10 @@ func ProvideChecker(
 		checker.PwHistoryDays = cfg.Policy.HistoryDays
 		checker.PasswordHistoryEnabled = cfg.Policy.IsEnabled()
 	}
+	if !*featureCfg.Password.Expiry.ForceChange.Disabled {
+		checker.PwExpiryForceChangeEnabled = cfg.Expiry.ForceChange.IsEnabled()
+		checker.PwExpiryForceChangeSinceLastUpdateInDays = cfg.Expiry.ForceChange.SinceLastUpdateInDays
+	}
```

Comment:
Remove changes to authn/authenticator/password/checker.go and authn/authenticator/password/deps.go